packages/app/src/webapp/cli.ts

```diff
@@ -138,7 +138,7 @@ const handleQueuedInput = async (nextBlock: HTMLElement) => {
   const queuedInput = disableInputQueueing()
 
   if (!queuedInput) {
-    return
+
```

Comment:
I believe we can do better here

Reply:
Sure, but as the first comment says, it was fixed automatically with the ESLint `--fix` option.

Anyway, updated!